crates/cli/src/state.rs

```diff
@@ -20,6 +21,8 @@ pub struct State {
     pub manifest: PackageManifest,
     /// Data from the `pnpm-lock.yaml` file.
     pub lockfile: Option<Lockfile>,
+    /// resolved packages
```

Comment:
This documentation is unhelpful. You may copy the (would be) documentation from the type alias to here.